src/background/adblocker.js

```diff
@@ -33,6 +33,8 @@ import Config, {
 
 let options = Options;
 
+const WEB_ACCESSIBLE_RESOURCES = new Set(__RESOURCES__);
```

Comment:
you can get this from `chrome.runtime.getManifest().web_accessible_resources` instead

Reply:
updated in f6016173db244d71cc6b1aff0aa927d0eb33c91b